cfme/tests/infrastructure/test_advanced_search_providers.py

```diff
@@ -44,112 +43,118 @@ def advanced_search_view():
-    test_filter_save_cancel.filter_name = fauxfactory.gen_alphanumeric()
-    logger.debug('Set filter_name to: {}'.format(test_filter_save_cancel.filter_name))
+    test_provider_filter_save_cancel.filter_name = fauxfactory.gen_alphanumeric()
+    logger.debug('Set filter_name to: {}'.format(test_provider_filter_save_cancel.filter_name))
 
     # Try save filter
     assert advanced_search_view.entities.search.save_filter(
         "fill_count(Infrastructure Provider.VMs, >)",
-        test_filter_save_cancel.filter_name, cancel=True)
+        test_provider_filter_save_cancel.filter_name, cancel=True)
     advanced_search_view.flash.assert_no_error()
 
     assert advanced_search_view.entities.search.reset_filter()
     # Exception depends on system state - Load button will be disabled if there are no saved filters
     with pytest.raises(NoSuchElementException):
         advanced_search_view.entities.search.load_filter(
-            saved_filter=test_filter_save_cancel.filter_name)
+            saved_filter=test_provider_filter_save_cancel.filter_name)
 
 
-def test_filter_save_and_load(rails_delete_filter, advanced_search_view):
+def test_provider_filter_save_and_load(rails_delete_filter, advanced_search_view):
     # bind filter_name to the function for fixture cleanup
-    test_filter_save_and_load.filter_name = fauxfactory.gen_alphanumeric()
-    logger.debug('Set filter_name to: {}'.format(test_filter_save_and_load.filter_name))
+    test_provider_filter_save_and_load.filter_name = fauxfactory.gen_alphanumeric()
+    logger.debug('Set filter_name to: {}'.format(test_provider_filter_save_and_load.filter_name))
 
     # Save filter
     assert advanced_search_view.entities.search.save_filter(
         "fill_count(Infrastructure Provider.VMs, >, 0)",
-        test_filter_save_and_load.filter_name)
+        test_provider_filter_save_and_load.filter_name)
     advanced_search_view.flash.assert_no_error()
 
     # Reset filter
     assert advanced_search_view.entities.search.reset_filter()
 
     # Load filter
-    assert advanced_search_view.entities.search.load_filter(test_filter_save_and_load.filter_name)
+    assert advanced_search_view.entities.search.load_filter(
+        test_provider_filter_save_and_load.filter_name
+    )
     advanced_search_view.flash.assert_no_error()
 
 
-def test_filter_save_and_cancel_load(rails_delete_filter, advanced_search_view):
+def test_provider_filter_save_and_cancel_load(rails_delete_filter, advanced_search_view):
     # bind filter_name to the function for fixture cleanup
-    test_filter_save_and_cancel_load.filter_name = fauxfactory.gen_alphanumeric()
-    logger.debug('Set filter_name to: {}'.format(test_filter_save_and_cancel_load.filter_name))
+    test_provider_filter_save_and_cancel_load.filter_name = fauxfactory.gen_alphanumeric()
+    logger.debug('Set filter_name to: {}'.format(
+        test_provider_filter_save_and_cancel_load.filter_name)
+    )
```

Comment:
You know I much prefer putting the closing parens down here at the indentation level of the call (i.e. I like that you do that.   There is so much code that does the opposite:
   
`some_func(
    blah,
    blather)'

That I thought the way you were doing it was not our standard.